docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,49 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and token usage for Ubuntu Pro
+*********************************************
+
+Ubuntu Pro tokens are used to authenticate your machine and enable access to the services you are entitled to. These tokens are unique and specific to your account, so you should treat them as a secret (as you would with a password).
+
+In this page, we'll discuss how Ubuntu Pro tokens work and how machines are counted against your subscription.
+
+Ubuntu Pro token(s)
+--------------------
+
+- Obtaining Pro tokens:
+     After subscribing to Ubuntu Pro, you can access your subscriptions through the `Ubuntu Pro Dashboard <Pro_dashboard_>`_. Each subscription includes a Ubuntu Pro token that can be used to attach as many machines as the subscription allows.
```

Comment:
Just looking at the rendered docs, the indent seems quite large. I think you only need two spaces worth of indentation for a bullet point in rst